tests/resources/rebuild_example_data

```diff
@@ -112,8 +101,13 @@ def main(ctx, rebuild_input):
         # The environ here should be the same (modulo the tempdir location)
         # as we use in test_examples.py.
         environ = {'ZIPLINE_ROOT': d.getpath('example_data/root')}
+
         if rebuild_input:
-            rebuild_input_data(environ)
+            raise StandardError(
```

Comment:
Maybe `NotImplementedError`? My preference is definitely for more specific error types when possible.